sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/QuantileSummariesSuite.scala

```diff
@@ -55,7 +55,7 @@ class QuantileSummariesSuite extends SparkFunSuite {
   }
 
   private def checkQuantile(quant: Double, data: Seq[Double], summary: QuantileSummaries): Unit = {
-    val approx = summary.query(quant)
+    val approx = summary.query(quant).get
```

Comment:
Add a test case with `summary.count == 0` and improve this helper function to cover it?